src/test/java/GmailClientTest.java

```diff
@@ -119,14 +124,31 @@ public void getQueryStringFromAnySender() {
     Assert.assertEquals(fromAnySenderQuery, EMPTY_QUERY);
   }
 
+  @Test
+  public void getQueryStringImportantEmailsOnly() {
+    String importantQuery = GmailClient.isImportantQuery(RETURN_IMPORTANT_ONLY);
```

Comment:
Nit: I think `true` would be clearer

Reply:
Just for consistency, I think it might be better to continue using constants for every value (we agreed to this in a standup a while back).

I don't mind changing this though if you have strong feelings either way. Please let me know!

Reply:
Raw booleans are already in a way constants mapping true/false to 1/0. I think it'd be fair to use them instead of custom constants even if we are using constants "for every value". But if you do want to define custom ones, maybe rename them so `isImportantQuery(RETURN_IMPORTANT_ONLY)` becomes more "readable"?

Reply:
Fair enough - for booleans I'll just use the values.